app/(backend)/factories/serviceFactory.ts

```
@@ -0,0 +1,83 @@
+  static getApiKeyService(): IApiKeyService {
+    if (!this.apiKeyService) {
+      this.apiKeyService = new ApiKeyService();
+    }
+    return this.apiKeyService;
+  }
+  
+  /**
+   * Get ModelService instance (singleton)
+   */
+  static getModelService(): IModelService {
+    if (!this.modelService) {
```

Biome (1.9.4)

[error] Using this in a static context can be confusing.

this refers to the class.
Unsafe fix: Use the class name instead.


(lint/complexity/noThisInStatic)